### PR TITLE
Enable partial unstake

### PR DESCRIPTION
**Pull request by mattlockyer**

### CODE REVIEW

File: src/components/staking/components/StakingAction.js
Comment (mattlockyer) [inline src/components/staking/components/StakingAction.js:37]: this could be a const outside of StakingAction function (module level after imports) and value could be `new BN(parseNearAmount('0.00001'))` then replace usage with DECIMAL_DEVIATION

File: src/components/staking/components/Stake.js
Comment (mattlockyer) [inline src/components/staking/components/Stake.js:27]: maybe we want to add this const somewhere and just import it?

```
const STAKING_AMOUNT_DEVIATION = new BN(parseNearAmount('0.00001'))
```

**Review by kcole16 — APPROVED** [review]

**Review by Patrick1904 — APPROVED** [review]

**Comment by Patrick1904** [comment]
@vgrichina I will pull in latest master to https://github.com/near/near-wallet/pull/1044 once this is merged

**Comment by vgrichina** [comment]
@Patrick1904 please don't wait for me to merge to `master` when you approved and sure stuff works alright!

**Pull request merged**
